### PR TITLE
COMMERCE-11891 - Automate test for COMMERCE-11737

### PR DESCRIPTION
**Pull request by jaydawu**
https://liferay.atlassian.net/browse/COMMERCE-11891

### CODE REVIEW

**Comment by jaydawu** [comment]
ci:test:sf

**Comment by jaydawu** [comment]
ci:test:commerce-functional

**Review by lorenzolapescara — CHANGES_REQUESTED** [review]

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceOrders.testcase [hunk 1]
```diff
@@ -3050,6 +3050,84 @@ definition {
...
+				jsonPath = "$.printedNote",
+				orderId = ${orderId},
+				parameter = "fields",
+				parameterValue = "printedNote",
+				password = "test");
+
+			TestUtils.assertEquals(
+				actual = ${printedNote},
+				expected = "Test Print Note Content");
+		}
```
[the remark this answers is not in the document]
Comment (jaydawu) [reply modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceOrders.testcase:3128]: Hi @lorenzolapescara, thanks for the reminder. I need to create a Minium site to see if the checkout is actually placed, so I updated the automated test accordingly.

**Comment by jaydawu** [comment]
ci:test:sf

**Comment by jaydawu** [comment]
ci:test:commerce-functional

**Review by lorenzolapescara — APPROVED** [review]

**Comment by jaydawu** [comment]
cc/ [victoroscuro](https://github.com/victoroscuro)

**Comment by lorenzolapescara** [comment]
Results are good, forwarding.

**Comment by lorenzolapescara** [comment]
ci:forward

**Pull request closed without merging**